<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntityWindow</class>
 <widget class="QDialog" name="EntityWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="entityListView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="showPushButton">
           <property name="text">
            <string>Afficher</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addPushButton">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modifyPushButton">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePushButton">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savePushButton">
           <property name="text">
            <string>Sauvegarder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadPushButton">
           <property name="text">
            <string>Charger</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::SaveAll</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EntityWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EntityWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
